xml/ha_storage_protection.xml

```diff
@@ -782,14 +842,49 @@ Timeout (msgwait)  : 10
     node needs to be fenced. </para>
   </sect1>
 
-<!--  <sect1 xml:id="sec.ha.storage.protect.fencing">
-  <title>Storage-based Fencing</title>
-  <para>
-   You can reliably avoid split brain scenarios by using one or more
-   &stonith; Block Devices (SBD), <literal>watchdog</literal> support and
-   the <literal>external/sbd</literal> &stonith; agent.
-  </para>
- </sect1>-->
+  <sect1 xml:id="sec.ha.storage.protect.test">
+   <title>Testing SBD and Fencing</title>
+   <para>To test if the combination of SBD and fencing works, use one or all
+    of the following methods to check them:
+   </para>
+   <itemizedlist>
+    <listitem>
+     <para>Manually trigger fencing for node <replaceable>NODENAME</replaceable>:</para>
+     <screen>&prompt.root;<command>crm</command> node fence <replaceable>NODENAME</replaceable></screen>
+    </listitem>
+    <listitem>
+     <para>Simulate a SBD failure on the SBD inquisitor and use its process ID:</para>
+     <screen>&prompt.root;<command>systemctl</command> status sbd
+● sbd.service - Shared-storage based fencing daemon
+
+   Loaded: loaded (/usr/lib/systemd/system/sbd.service; enabled; vendor preset: disabled)
+   Active: active (running) since Tue 2018-04-17 15:24:51 CEST; 6 days ago
+     Docs: man:sbd(8)
+  Process: 1844 ExecStart=/usr/sbin/sbd $SBD_OPTS -p /var/run/sbd.pid watch (code=exited, status=0/SUCCESS)
+ Main PID: 1859 (sbd)
+    Tasks: 4 (limit: 4915)
+   CGroup: /system.slice/sbd.service
+           ├─<emphasis role="strong">1859 sbd: inquisitor</emphasis>
+[...]</screen>
+     <para>Use the process ID (in our example, <literal>1859</literal>) and
+      kill the SBD inquisitor:
+     </para>
+     <screen>&prompt.root;<command>kill</command> -9 1859 </screen>
+    </listitem>
+    <listitem>
+     <remark>toms 2018-04-24: This is a bit vague and I'm not entirely sure
+      about this step. Taken from FATE entry, developers could add more info.
+     </remark>
+     <para>
+      Produce a failure of resource operation. This failure triggers a
+      fencing action. For example, produce a failure of resource stop
```

Comment:
1. Configure a ```<property>on-fail=fence</property>``` property  for a  resource monitor operation: [...] (not sure if giving the line is enough, perhaps also mention command with which to set this property?) 
2. Produce a failure of the monitoring operation (how???).
This failure triggers a fencing action (as defined above).

Reply:
I'm not actually quite sure if we should provide this way for simulating fencing. Although it's convenient and works for testing, it's not a typical scenario ;)